contrib/terraform/openstack/modules/compute/main.tf

```diff
@@ -19,12 +19,13 @@ resource "openstack_networking_secgroup_rule_v2" "k8s_master" {
 }
 
 resource "openstack_networking_secgroup_v2" "bastion" {
-  name        = "${var.cluster_name}-bastion"
+  count       = "${var.number_of_bastions ? 1 : 0}"
+  name        = "${format("%s-bastion", var.cluster_name)}"
```

Comment:
Is the use of `format()` really necessary, most places use `"${var.foo}-bar"` in the rest of the TF files

Reply:
see below